config/pools.yaml

```diff
@@ -48,9 +48,23 @@
 ##      # TODO: The atomicity specification needs to be polished.
 ##      cmd_delete: "/bin/terminate-vm"
 ##
-##      # List of tags which any resource coming from this pool will have.  Based on
-##      # this, resources are assigned to user's tickets.
+##      # List of tags which any resource coming from this pool will have.
+##      # Based on these tags, resources are assigned to user's tickets.
+##      # Different pools can provide the same kind of resources (same tags).
+##      # To prefer resources from one pool over another, there's a possibility
+##      # to specify a priority value for each pool tag (default 0).  Resalloc
+##      # server then, when assigning resources to tags, prefers resources with
+##      # higher tag priority score.  See on example:
+##      #  - when a preallocated resource A has tags foo=0, bar=1, baz=2, and
```

Comment:
The `foo=0`, `bar=1` notation is confusing. I understood the whole paragraph above, then got unsure when reading the example and realized that I understood it correctly when finished reading the tags: section. I would probably reword it more explicitly, e.g.

- when a preallocated resource A has tags foo (priority 0), bar (priority 1), baz (priority 2), and

The `foo=0` notation gives the impression that the syntax would be

```
tags:
  - foo=0
  - bar=1
```